.github/pull_request_template.md

```diff
@@ -0,0 +1,98 @@
+## Pull Request Template
+
+### Overview
+
+The MSDK repo follows a custom rule-set based on the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard. The pull request titles will follow this format:
+
+`type(scope): Subject`
+
+#### Types
+
+The list of possible types are:
+
+1.  **fix** – a bug fix has occurred
+2.  **feat** – a new feature was introduced
+3.  **chore** – changes that do not relate to a fix or feature, or a catch-all type for changes that don’t really fit under any of the other types
+4.  **refactor** – refactored code that neither fixes a bug nor adds a new feature
+5.  **docs** – updates to documentation
+6.  **style** – code styling changes (linter or clang-format)
+7.  **test** – changes involving test code (not changes like deleting a line of code to test if your application works or not)
+8.  **perf** – performance improvements
+9.  **ci** – changes related to continuous integration
+10. **build** – changes that affect build system, tool chain, or any other external dependencies
+11. **revert** – reverts a commit
+
+Appending an `!` at the end of the type signifies the PR is a breaking or major change.
+
+NOTE: The type is case sensitive and must match one of the listed types.
+
+#### Scopes
+
+The scopes are dependent on the changes based on their location in the MSDK. This list of possible scopes are:
+
+1.  **BLE** – Any bluetooth related changes (e.g. Cordio)
+2.  **Documentation** – Any changes in msdk/Documentation folder
+3.  **Examples** – Any changes in msdk/Examples
+4.  **Tools** – Any changes in msdk/Tools
+5.  **Boards** – Any board file changes
+6.  **CMSIS** – Any CMSIS/Register file changes
+7.  **MiscDrivers** – Any MiscDrivers changes (e.g. TFT Display, PB, LED, External Flash Drivers)
+8.  **PeriphDrivers** – Any changes in msdk/Libraries/PeriphDrivers
+9.  **Third-Party** – All third-party libraries (e.g. FreeRTOS, LVGL, lwIP)
+10. **ignore** – Small and quick miscellaneous fixes (should not be used often)
+11. **workflow** – Any GitHub workflow related changes
+
+NOTE: The scope is case sensitive and must match one of the listed scopes.
+
+### Responsibilites for Reviewers/Mergers/PR Owners
```

Comment:
I suggest moving this into CONTRIBUTING.md.  It makes the PR template too cluttered/long.  Keep in mind that users will need to delete these types of sections from the PR every time